### ISSUE TITLE
Add `Application.SetHighDpiMode(HighDpiMode.SystemAware);` for WinForms Project Upgrade

### ISSUE THREAD

**Issue by myokeeh** [dotnet/upgrade-assistant#545]
### Describe the bug
Upgraded WinForms projects may not scale as expected after upgrade-assistant.

### Solution:
I added `Application.SetHighDpiMode(HighDpiMode.SystemAware);` to have my app look more or less the same.

```
[STAThread]
static void Main()
{
    Application.SetHighDpiMode(HighDpiMode.SystemAware);
    Application.EnableVisualStyles();
    Application.SetCompatibleTextRenderingDefault(false);
    Application.Run(new PhotoDateChanger());
}
```

See https://github.com/dotnet/winforms/issues/4947

**Comment by RussKie** [comment]
One should be using `Application.SetHighDpiMode(HighDpiMode.PerMonitorV2);`

/cc: @OliaG @dreddy-work

**Comment by sunandabalu** [comment]
We will watch for how dotnet/winforms-designer#3278 plays out.

**Comment by myokeeh** [comment]
> We will watch for how [dotnet/winforms-designer#3278](https://github.com/dotnet/winforms-designer/issues/3278) plays out.

Showing 404 for me.

**Comment by RussKie** [comment]
This is a non-public repo.

**Comment by sunandabalu** [comment]
This is addressed by fix for #785 , closing this issue
